spec/requests/graphql/queries/users/get_single_user_spec.rb

```diff
@@ -9,12 +9,27 @@
       result = JSON.parse(response.body)
 
       expect(result.dig('data',
-                        'getUser')).to eq({
-                                            'id' => user.id.to_s,
-                                            'name' => user.name,
-                                            'email' => user.email,
-                                            'birthdate' => user.birthdate.to_s
-                                          })
+                        'getUser')).to include({
+                                                 'id' => user.id.to_s,
+                                                 'name' => user.name,
+                                                 'email' => user.email,
+                                                 'birthdate' => user.birthdate.to_s
+                                               })
+
+      expect(result['data']['getUser']['eras']).to eq(
+        user.eras.map do |era|
+          {
+            'id' => era.id.to_s,
+            'userId' => era.user_id.to_s,
+            'name' => era.name.to_s,
+            'startDate' => era.start_date.to_s,
+            'endDate' => era.end_date.to_s,
+            'color' => era.color,
+            'createdAt' => era.created_at.to_s,
+            'updatedAt' => era.updated_at.to_s
+          }
```

Comment:
I tried adding all assertions in line 19-30 to the above block after line 16 but kept getting syntax errors. Since it is looking at so many fields within eras I thought this was also a bit more readable anyway.

Reply:
Yeah I think that was a good move.  Great test Leah.